element_array_ephys/ephys_acute.py

```diff
@@ -140,57 +154,77 @@ def auto_generate_entries(cls, session_key):
         Method to auto-generate ProbeInsertion entries for a particular session
         Probe information is inferred from the meta data found in the session data directory
         """
-        session_dir = find_full_path(get_ephys_root_data_dir(),
-                                  get_session_directory(session_key))
+        session_dir = find_full_path(
+            get_ephys_root_data_dir(), get_session_directory(session_key)
+        )
         # search session dir and determine acquisition software
-        for ephys_pattern, ephys_acq_type in zip(['*.ap.meta', '*.oebin'],
-                                                 ['SpikeGLX', 'Open Ephys']):
+        for ephys_pattern, ephys_acq_type in zip(
+            ["*.ap.meta", "*.oebin"], ["SpikeGLX", "Open Ephys"]
```

Comment:
You don't need the zip here is you know the lists:
```suggestion
        for ephys_pattern, ephys_acq_type in (
            ("*.ap.meta", "SpikeGLX") ("*.oebin", "Open Ephys")
```